packages/site-kit/package.json

```diff
@@ -23,19 +22,79 @@
 		"svelte-local-storage-store": "^0.4.0"
 	},
 	"devDependencies": {
-		"@sveltejs/kit": "^1.3.10",
-		"@sveltejs/package": "^1.0.2",
+		"@sveltejs/kit": "^1.15.5",
+		"@sveltejs/package": "^2.0.2",
 		"flexsearch": "^0.7.31",
-		"svelte": "^3.55.1",
-		"typescript": "^4.9.5",
-		"vite": "^4.0.4"
+		"svelte": "^3.58.0",
+		"typescript": "^5.0.4",
+		"vite": "^4.2.1"
 	},
 	"publishConfig": {
-		"access": "public",
-		"directory": "package"
+		"access": "public"
 	},
 	"peerDependencies": {
 		"@sveltejs/kit": "^1.0.0",
 		"svelte": "^3.54.0"
-	}
+	},
+	"exports": {
+		"./actions": {
+			"types": "./dist/actions/index.d.ts",
+			"default": "./dist/actions/index.js"
+		},
+		"./branding/svelte-logo.svg": "./dist/branding/svelte-logo.svg",
+		"./components/home": {
```

Comment:
Do we need deep exports like this or can they all come from `./component`, `./search` etc?

Reply:
It's just naming. Ig i can put this one out or components like theme

Reply:
What I mean is: export `TryTerminal` from `./components` instead; and remove all the search deep exports so there's only `./search` left.

Reply:
Theme relies on a store. We can put that in script context module in the component though. It could look messy though

Reply:
Re: TryTerminal: Reason its in its own folder is because it is used only on home, whereas the rest of the components are generic kind and used in many places

Reply:
Moved it all to the root